README.md

```diff
@@ -1,43 +1,48 @@
-# CoderBabez Homework 01 - HTML Basics
-
-### OBJECTIVES
-
-*	Practice forking a github repository, cloning it to your computer, committing changes and publishing to gh-pages
-* Practice your foundation HTML tags 
-* Practice basic content tags - h1, h2, p, img
-* Learn two new tags - ordered and unordered lists
-
-
-### Description:
-
-Your Grandma wants you to publish her famous cookie recipe to the world wide web, she's typed it out for you in this .html document (she's so nice). 
-She would like....
-* The name of her recipe to appear big at the top of the page
-* The directions to appear in order in a numbered list.
-* The ingredients to appear in a bulleted list
-* A picture of her famous cookies on the top of the page (hint: you will need to link to a relative url in the img folder of this repository. Your url will look  like this "./img/cookies.jpg")
-* Nutrition facts in a paragraph below the recipe
-* A link to her inspiration recipe at the bottom of the page
-
-
-#### Content: 
-Use the following content for the recipe: 
-```
-The Best Chocolate Chip Cookies
-Recipe by Grandma
-Prep Time: 45 Min
-
-INGREDIENTS
-1 1/2 cup (3 sticks) softened butter, 1 cup brown sugar, 1 cup granulated sugar, 1 Tbl vanilla instant pudding powder, 2 Tbl milk, 2 Tbl vanilla extract, 2 eggs, 4 cups all purpose flour, 2 tsp baking soda, 1/2 tsp salt, 4 cups chocolate chips, 1 cup chopped walnuts or pecans(optional)
-
-How to make Mama’s Recipe: 
-Preheat oven to 350 degrees Beat butter and sugars together until light and fluffy. Stir in pudding mix milk and vanilla extract. Beat in eggs. Add dry ingredients and stir until combined.
-Stir in chocolate chips and nuts.Place 1 1/2 inch balls of dough 2 inches apart on an un greased cookie sheet. Bake 8-10 minutes or until golden brown.
-
-
-Nutrition Information
-
-Probably bad for you!
-
-Inspiration Recipe: "http://www.opensourcefood.com/people/Amanori/recipes/mamas-recipe-the-best-chocolate-chip-cookies"
-```
+<article>
+<header>
+<h1> The Best Chocolate Chip Cookies Recipe </h1>
+<h3>by Grandma</h3>
+<p>Prep Time: 45 Min</p>
+</header>
+<br>
+<img src="./img/cookies.jpg" alt=Chocolate Chip Cookies>
+<br>
+<h2>INGREDIENTS</h2>
+<ul style="list-style-type:disc;">
+<li>1 1/2 cup (3 sticks) softened butter</li>
+<li>1 cup brown sugar</li>
+<li>1 cup granulated sugar</li>
+<li>1 Tbl vanilla instant pudding powder</li>
+<li>2 Tbl milk</li>
+<li>2 Tbl vanilla extract</li>
+<li>2 eggs</li>
+<li>4 cups all purpose flour</li>
+<li>2 tsp baking soda</li>
+<li>1/2 tsp salt</li>
+<li>4 cups chocolate chips</li>
+<li>1 cup chopped walnuts or pecans(optional)</li>
+</ul>
+<br>
+<br>
+<br>
```

Comment:
<section> tags will help organize and break up sections, so you wouldn't need to add multiple break tags to create space